res/css/structures/_ToastContainer.scss

```diff
@@ -122,12 +123,11 @@ limitations under the License.
                 vertical-align: middle;
             }
 
-            span {
-                padding-left: 8px;
-                float: right;
```

Comment:
this might have been deliberate, even in a RTL language - will have to check with design.

Reply:
Unless this would be expected to cover something in addition to `<span>{ countIndicator }</span>`, this should work.